src/devices/steelmate.c

```diff
@@ -87,12 +87,12 @@ static int steelmate_callback(bitbuffer_t *bitbuffer) {
 
 		data = data_make("time", "", DATA_STRING, time_str,
 			"type", "", DATA_STRING, "TPMS",
-			"make", "", DATA_STRING, "Steelmate",
+			"model", "", DATA_STRING, "Steelmate",
 			"id", "", DATA_STRING, sensorIDhex,
 			"pressure_PSI", "", DATA_DOUBLE, pressurePSI,
 			"temperature_F", "", DATA_DOUBLE, (float)tempFahrenheit,
 			"battery_mV", "", DATA_INT, battery_mV,
-			"checksum", "", DATA_STRING, "OK",
+			"crc", "", DATA_STRING, "OK",
```

Comment:
This really is a checksum. Naming it crc is wrong. Maybe change all instances to message integrity check instead (mic) ?